scripts/makeUserAdmin.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * You can run this script like this:
+ * NODE_ENV=develop USER_ADDRESS=0x5AC583Feb2b1f288C0A51d6Cdca2e8c814BFE93B node scripts/makeUserAdmin.js
+ */
+
+const config = require('config');
+const mongoose = require('mongoose');
+
+const userAddress = process.env.USER_ADDRESS;
```

Comment:
Please get user admin as an argument passed to script, like getPledge and getPledgeAdmin scripts

Reply:
@aminlatifi  done